packages/calypso-products/src/products-list.ts

```diff
@@ -432,6 +440,40 @@ export const JETPACK_SITE_PRODUCTS_WITH_FEATURES: Record<
 		getProductId: () => 2400,
 		getStoreSlug: () => PRODUCT_JETPACK_BOOST_MONTHLY,
 	},
+
+	[ PRODUCT_JETPACK_MONITOR_YEARLY ]: {
+		product_name: translate( 'Monitor' ),
```

Comment:
As per the design, the title is `Downtime Monitoring.` I might be missing something here.